src/components/labware/Labware.tsx

```diff
@@ -381,24 +408,27 @@ const Labware = ({
       <div onClick={() => onClick?.()} className={labwareClasses}>
         {(isBarcodeInfoAtTheLeftSide || isBarcodeInfoAtTheTop) && BarcodeInformation()}
         <div className={gridClasses}>
-          {buildAddresses({ numColumns, numRows }, gridDirection).map((address, i) => (
-            <Slot
-              key={i}
-              address={address}
-              slot={slotByAddress[address]}
-              size={numColumns > 2 ? 'medium' : numColumns > 6 || numRows > 6 ? 'small' : 'large'}
-              onClick={internalOnClick}
-              onCtrlClick={onCtrlClick}
-              onShiftClick={onShiftClick}
-              onMouseEnter={onSlotMouseEnter}
-              onMouseLeave={onSlotMouseLeave}
-              text={slotText}
-              secondaryText={slotSecondaryText}
-              color={_slotColor}
-              selected={selectedAddresses?.has(address) || (highlightedSlots?.has(address) ?? false)}
-              isCleanedOut={cleanedOutAddresses?.includes(address)}
-            />
-          ))}
+          {buildAddresses({ numColumns, numRows }, gridDirection).map((address, i) => {
+            return (
+              <div key={address} className={`p-1 ${slotSectionBgColor()[address]} rounded-lg transition`}>
+                <Slot
+                  address={address}
+                  slot={slotByAddress[address]}
+                  size={numColumns > 2 ? 'medium' : numColumns > 6 || numRows > 6 ? 'small' : 'large'}
```

Comment:
I don't think the logic for this conditional expression is right. It results in "medium" whenever numColumns > 2, and if it's not greater than 2, it then checks if numColumns > 6, which would be impossible. Maybe it's supposed to be something like `numColumns <= 2 ? 'large' : numColumns <= 6 && numRows <= 6 ? 'medium' : 'small'.